lib/config/presets/internal/custom-managers.ts

```diff
@@ -90,11 +90,11 @@ export const presets: Record<string, Preset> = {
     customManagers: [
       {
         customType: 'regex',
-        fileMatch: [
-          '(^|/)Makefile$',
-          '(^|/)makefile$',
-          '(^|/)GNUMakefile$',
-          '\\.mk$',
+        filePatterns: [
+          '/(^|/)Makefile$/',
+          '/(^|/)makefile$/',
+          '/(^|/)GNUMakefile$/',
+          '/\\.mk$/',
```

Comment:
```suggestion
          '**/Makefile',
          '**/GNUMakefile',
          '**/*.mk',
```

Note: glob is case-insensitive so we don't need Makefile and makefile